app/calculations.py

```diff
@@ -231,7 +229,8 @@ def calculate_crops_prod_df(all_params, years):
     return crops_prod_df
 
 
-def calculate_yield_df(all_params, years):
+def calculate_yield_df(all_params, years, land_use):
```

Comment:
By explicitely passing `land_use` to this function we make it clear that it needs this data.